src/identity/email-password-identity-provider/default-views/_layout.ejs

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html >
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Sign-in</title>
+    <style>
```

Comment:
Let's not have any inline CSS styling (and no external dependencies like Google Fonts).
We need to keep the hypertext markup and styling separate.

Styles will be served by (customizable) static file handlers.

Reply:
Okay will do. I'll put all styles in a static file handler.